src/static/index.html

```diff
@@ -13,6 +13,14 @@
           rel="stylesheet" type="text/css">
     <link href="assets/css/markdown.css"
           rel="stylesheet" type="text/css">
+    <!-- IGV and its dependencies -->
+    <link rel="stylesheet" type="text/css" href="https://ajax.googleapis.com/ajax/libs/jqueryui/1.11.2/themes/smoothness/jquery-ui.css"/>
+    <link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css">
+    <link rel="stylesheet" type="text/css" href="https://igv.org/web/release/1.0.1/igv-1.0.1.css">
+    <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
+    <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.11.2/jquery-ui.min.js"></script>
+    <script type="text/javascript" src="https://igv.org/web/release/1.0.1/igv-1.0.1.js"></script>
```

Comment:
let's use the minified version - https://igv.org/web/release/1.0.1/igv-1.0.1.min.js - 377K vs 805K.